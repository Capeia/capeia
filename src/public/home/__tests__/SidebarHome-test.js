// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<SidebarHome>', () => {
  it('renders correctly', () => {
    jest.mock('shared/DeferredDataProvider')
    const Component = require('../SidebarHome').default
    const tree = renderer.create(<Component />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
