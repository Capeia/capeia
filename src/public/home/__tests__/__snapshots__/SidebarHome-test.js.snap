// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SidebarHome> renders correctly 1`] = `
<div>
  <section
    className="intro"
  >
    <h2
      style={
        Object {
          "fontSize": "1.8em",
        }
      }
    >
      Capeia is a new way to interact with science.
    </h2>
    <ul
      className="sellingPoints"
    >
      <li
        className={undefined}
      >
        <span>
          <i>
            Discover
          </i>
           new developments in cutting-edge sciences.
        </span>
      </li>
      <li
        className={undefined}
      >
        <span>
          <i>
            Get in touch
          </i>
           with people involved in the field.
        </span>
      </li>
      <li
        className={undefined}
      >
        <span>
          <i>
            Support
          </i>
           research that is meaningful to you.
        </span>
      </li>
    </ul>
    <p>
      <a
        className="btn btn-capeia btn-lg highlight"
        href="/welcome"
      >
        Take the Tour
      </a>
    </p>
  </section>
  <section>
    <div>
      &lt;DeferredDataProvider&gt;
        &lt;Relay(AotmBox) /&gt;
      &lt;/DeferredDataProvider&gt;
    </div>
  </section>
</div>
`;
